packages/components/organisms/f-registration/src/components/_tests/Registration.test.js

```diff
@@ -188,7 +192,7 @@ describe('Registration', () => {
 
             it('should populate generic error message and emit failure event when service responds with a 400', async () => {
                 // Arrange
-                const err = { response: { data: { faultId: '123', traceId: '123', errors: [{ description: 'The Password field is required', errorCode: '400' }] } } };
+                const err = { response: { status: 400, data: { faultId: '123', traceId: '123', errors: [{ description: 'The Password field is required', errorCode: '400' }] } } };
```

Comment:
Minor, but we're formatting differently this test and the one that you modified above (one line vs multiple lines).

Reply:
Yup I didn't write them, just amended one property; have formatted the whole file so it should be consistent, but I don't want to colour my PR with tons of files with formatting changes.